styles.css

```
@@ -0,0 +1,25 @@
+@import url('https://fonts.googleapis.com/css2?family=Baloo+Tamma+2&display=swap');
+
+body {
+    background-color: #0e1016;
+}
+
+main {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+}
+
+.title {
+    color: #F4F3EA;
+    text-align: center;
+    font-family: 'Baloo Tamma 2', cursive;
```

Curti a escolha de fonte principal, bonita e simples. Sobre as fontes 'seguras', sugiro de continuarmos com as anteriores. 
1 - pq `cursive` é muito diferente da principal, então é perceptível a mudança de estilo enquanto a fonte principal está sendo baixada.
2 - esse segundo argumento é completamente arbitrário, só por motivos de gosto mesmo haha. Acho `cursive` meio breguinha.

```suggestion
    font-family: 'Baloo Tamma 2', Helvetica,sans-serif;
```

---

Concordo contigo que a cursive não e a melhor opção, mas eu não me liguei porque importei do Google Fonts e a font family veio com esse padrão. Depois que eu vi seu comentário que fui reparar nisso, rs.

---

Sugestão aplicada :D